weave-js/src/components/PagePanelComponents/Home/Browse3/datasets/CellRenderers.tsx

```diff
@@ -44,6 +44,7 @@ interface CellViewingRendererProps {
   isNew?: boolean;
   isEditing?: boolean;
   serverValue?: any;
+  disableNewRowHighlight?: boolean;
```

Comment:
I thought the green highlight for "added rows" ruined this view, so I added a prop to disable.